src/content/it/fundamentals/performance/critical-rendering-path/render-tree-construction.md

```diff
@@ -2,82 +2,153 @@ project_path: /web/_project.yaml
 
-{# wf_updated_on: 2014-09-17 #}
+{# wf_updated_on: 2017-07-14 #}
 {# wf_published_on: 2014-03-31 #}
 
 # Costruzione della struttura di rendering, layout e disegno {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
+Le strutture CSSOM e DOM vengono unite in una struttura di rendering, che viene
+poi utilizzata per calcolare il layout di ciascun elemento visibile e che serve
+da input al processo di disegno che esegue il rendering dei pixel sullo schermo.
+Ottimizzare ciascuno di questi passaggi è cruciale per ottenere una performance
+di rendering ottimale.
 
-Le strutture CSSOM e DOM vengono unite in una struttura di rendering, che viene poi utilizzata per calcolare il layout di ciascun elemento visibile e che serve da input al processo di disegno che esegue il rendering dei pixel sullo schermo. Ottimizzare ciascuno di questi passaggi è cruciale per ottenere una performance di rendering ottimale.
-
-
-Nella sezione precedente sulla costruzione del modello oggetto, abbiamo costruito le strutture DOM e CSSOM sulla base dell'input HTML e CSS. Tuttavia, entrambi sono oggetti indipendenti che possono registrare aspetti diversi del documento: uno descrive il contenuto e l'altro che regole di stile che devono esservi applicate. Come uniamo i due e portiamo il browser a eseguire il rendering dei pixel sullo schermo?
+Nella sezione precedente sulla costruzione del modello oggetto, abbiamo
+costruito le strutture DOM e CSSOM sulla base dell'input HTML e CSS. Tuttavia,
+entrambi sono oggetti indipendenti che possono registrare aspetti diversi del
+documento: uno descrive il contenuto e l'altro che regole di stile che devono
+esservi applicate. Come uniamo i due e portiamo il browser a eseguire il
+rendering dei pixel sullo schermo?
 
 ### TL;DR {: .hide-from-toc }
 - Le strutture DOM e CSSOM vengono unite per formare la struttura di rendering.
-- La struttura di rendering contiene solo i nodi richiesti per eseguire il rendering della pagina.
+- La struttura di rendering contiene solo i nodi richiesti per eseguire il
+  rendering della pagina.
 - Il layout calcola la posizione e dimensione esatta di ogni oggetto.
-- Il disegno è l'ultimo passaggio che include la struttura di rendering finale e ne esegue il rendering dei pixel sullo schermo.
+- Il disegno è l'ultimo passaggio che include la struttura di rendering finale
+  e ne esegue il rendering dei pixel sullo schermo.
 
 
-Il primo passaggio prevede che il browser unisca DOM e CSSOM in una 'struttura di rendering' che registra l'intero contenuto DOM visibile sulla pagina, oltre a tutte le informazioni di stile CSSOM per ciascun nodo.
+Il primo passaggio prevede che il browser unisca DOM e CSSOM in una 'struttura
+di rendering' che registra l'intero contenuto DOM visibile sulla pagina, oltre
```

Comment:
di rendering" che registra l'intero contenuto DOM visibile sulla pagina, oltre

Reply:
It seems no change.

Reply:
Same as above, just the quotes.